Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/OSIRISDiffractionReduction.py

```diff
@@ -257,8 +335,7 @@ def PyInit(self):
 
         # Note that dRange numbers are offset to match the numbering in the OSIRIS manual
         # http://www.isis.stfc.ac.uk/instruments/osiris/documents/osiris-user-guide6672.pdf
-        self.declareProperty('DRange', 1, validator=IntBoundedValidator(1, len(TIME_REGIME_TO_DRANGE) + 1),
-                             doc='Drange to use when DetectDRange is disabled')
+        self.declareProperty('DRange', defaultValue="", doc='Dranges to use when DetectDRange is disabled')
```

Comment:
update the doc string to describe what kind of input you need, ie a list of numbers